paper/paper.md

```diff
@@ -1,43 +1,204 @@
+
+When the number of observations in a dataset far exceeds the number of features,
+the estimator of choice for the covariance matrix is the sample covariance
+matrix. It is efficient under minimal regularity assumptions on the
+data-generating distribution. In high-dimensional regimes, however, its
+performance is unsatisfactory: the sample covariance matrix is highly variable,
+producing estimates with diverging condition numbers and over-dispersed
+eigenvalues [@johnstone2001distribution]. These issues amplify estimation error
+in resultant estimates, as well as in downstream analyses relying upon them.
+
+As high-dimensional data have become widespread, researchers have derived many
+novel covariance matrix estimators to remediate the sample covariance matrix's
+deficiencies. These estimators come in many flavours, though most are
+constructed by regularizing the sample covariance matrix. Comprehensive
+reviews of these estimators are provided by @fan2016 and @pourahmadi2013.
+
+This variety brings with it many challenges. Identifying an "optimal" estimator
+from among a collection of candidates can prove a daunting task, one whose
+objectivity is often compromised by the data analyst's decisions. Though
+data-driven approaches for selecting an optimal estimator from among estimators
+belonging to certain (limited) classes have been derived, the question of
+selecting from a diverse collection of candidate procedures remains unaddressed.
+
+# `cvCovEst` Framework
+
+The solution provided by `cvCovEst` a general, cross-validation-based,
+estimator-agnostic framework for covariance matrix estimator selection. The
+asymptotic optimality of selections are guaranteed under a few non-restrictive
+assumptions by extending the seminal work of @laan_dudoit:2003, @dudoit2005, and
+@vaart2006 on data-adaptive estimator selection to high-dimensional covariance
+matrix estimation [@boileau2021]. Here, optimality is defined as choosing an
+estimator with an equivalent risk difference to that which would have been
+selected were the underlying data-generating distribution _known_.
+
+The `cvCovEst` software package implements this framework for the `R` language
+and environment for statistical computing [@R]. Included is an accumulation of
+covariance matrix estimators spanning the work of many researchers (Table 1).
+They may be employed independently of the cross-validation procedure.
+`cvCovEst` also provides a slew of plotting and summary functions. These
+diagnostic tools allow users to gauge the algorithm's performance, diagnose
+issues that might arise during estimation procedures, and build intuition about
+the many estimators' behaviours. Additionally, users have options to increase
+the cross-validation method's computational efficiency via parallel
+computation. Parallelization relies on the suite of `future` packages
+[@future] by way of the `origami` package [@origami].
+
+Table 1: Covariance matrix estimators implemented as of version 0.3.4.
```

Comment:
```suggestion
Table 1: Covariance matrix estimators implemented as of [version 0.3.4](https://github.com/PhilBoileau/cvCovEst).
```
We should eventually replace the link to the package with one specifying a git tag / GitHub release of v0.3.4